### DISCUSSION TITLE
There is no BSD-3-Clause licence. How to add this in DT

### DISCUSSION THREAD

**Discussion by spmishra121** [DependencyTrack/dependency-track#2316] [Q&A] [answered]
I am seeing that lots of policy violation is coming because there is no license name as in example "BSD-3-Clause". Is there any way to add this?
![image](https://user-images.githubusercontent.com/6615148/208660936-61700c99-21f7-4dec-85b6-19a02c022310.png)

**Comment by syalioune** [comment, marked as answer]
DT already contains `BSD-3-Clause`. You can see it in `Home / Licenses`. The license is also correctly identified from a SBOM with npm package moo, see below.

![image](https://user-images.githubusercontent.com/6144741/209584758-f0ff0c91-8122-4857-b6f6-ca6a2b2d67a7.png)

To better adress your concern :

- What version of DT are you using ?
- What are you trying to achieve ? What's the current result and what were you expected precisely ? Screenshots are always welcome :smiley:
